add tenant admin clarification (#670)
MicrosoftDocs/OfficeDocs-O365ITPro: Office365-Admin/manage/centralized-deployment-of-add-ins.md

```powershell
Invoke-CompatibilityCheck
```

>    which prompts you for  *_TenantDomain_* (for example, *TailspinToysIncorporated.onmicrosoft.</span>com*) and  *_TenantAdmin_* credentials (Global admin credentials must be entered), and then requests consent.

Reviewer: ```suggestion
   which prompts you for  *_TenantDomain_* (for example, *TailspinToysIncorporated.onmicrosoft.</span>com*) and  *_TenantAdmin_* credentials (use your global admin sign in), and then requests consent.
```
